src/index.html

```diff
@@ -10,13 +10,54 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#"class="logo">
+        <img
+          src="/src/images/logo.png" alt="navigationlogo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="is-active nav__link">apple</a>
```

Comment:
The `href` attribute is missing in the anchor tag. It should point to the corresponding section of the page, e.g., `href="#apple"`.